spec/support/idv_examples/max_attempts.rb

```diff
@@ -1,78 +1,61 @@
-shared_examples 'idv max step attempts' do |sp|
-  it 'allows 3 attempts in 24 hours', :email do
-    visit_idp_from_sp_with_loa3(sp)
-    user = register_user
+shared_examples 'verification step max attempts' do |step|
+  scenario 'more than 3 attempts in 24 hours prevents further attempts' do
+    visit_idp_from_sp_with_loa3(:oidc)
 
-    max_attempts_less_one.times do
-      visit verify_session_path
-      fill_out_idv_form_fail
-      click_idv_continue
-
-      expect(current_path).to eq verify_session_result_path
+    if step == :phone
+      click_idv_begin
+      click_idv_address_choose_phone
     end
 
-    user.reload
-    expect(user.idv_attempted_at).to_not be_nil
-
-    fill_out_idv_form_fail
-    click_idv_continue
-
-    expect(page).to have_css('.alert-error', text: t('idv.modal.sessions.heading'))
-
-    visit_idp_from_sp_with_loa3(sp)
     expect(page).to have_content(
       t('idv.messages.hardfail', hours: Figaro.env.idv_attempt_window_in_hours)
     )
-    expect(current_url).to eq verify_fail_url
+    expect(current_url).to eq(verify_fail_url)
 
-    visit verify_session_path
-    expect(page).to have_content(t('idv.errors.hardfail'))
```

Comment:
I think there's one more expectation that needs to be added. Before, this was verifying that if the user tried to visit the `verify_session_path` directly while in the max attempts period, it would display the hardfail error. I don't see this scenario being tested in the refactor.

Reply:
The first example sort of covers that, but the user is coming from an SP. I can change it to try coming from an SP and visiting directly.

Reply:
Just pushed that. PTAL